ember-cli-build.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-env node */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
+  const app = new EmberAddon(defaults, {
```

Comment:
If this change is being caused by linting due to the new version of the rules (which it is) then I am fine with the change.  I was unaware that is how the change was coming to be and thought it was an non-required change.  We are currently upgrading all of the repos to `ember-cli 2.12.3` and had a discussion about the fact that this file specifically had a changeset to replace one instance of `var` with `const` but not another and we arrived at the outcome previously stated (https://github.com/ciena-blueplanet/ember-test-utils/pull/88#discussion_r125745077).  But that is because we are also not changing the version of `ember-test-utils` being used by the repos either - that will be a separate effort.